### PR TITLE
document HPA's implicit deactivation

### PR DESCRIPTION
**Pull request by novas0x2a**
I often find myself linking to the source code to explain this behavior, so I'm adding it to the docs :)

### CODE REVIEW

File: content/en/docs/tasks/run-application/horizontal-pod-autoscale.md [hunk 1]
```diff
@@ -433,7 +433,14 @@ behavior:
     selectPolicy: Disabled
 ```
 
-
+## Support for implicit maintenance-mode deactivation
+
+Starting from v1.4, you can implicitly deactivate the HPA for a target without the
```
Comment (sftim) [inline content/en/docs/tasks/run-application/horizontal-pod-autoscale.md:438]:
```suggestion
You can implicitly deactivate the HPA for a target without the
```
(we assume readers are using a supported version of Kubernetes)

File: content/en/docs/tasks/run-application/horizontal-pod-autoscale.md [hunk 2]
```diff
@@ -433,7 +433,14 @@ behavior:
...
+need to change the HPA configuration itself. If the target's desired replica count
+is set to 0, and the HPA's minimum replica count is greater than 0, the HPA will
+cease to adjust the target (and will set the `ScalingActive` Condition on itself
```
Comment (sftim) [inline content/en/docs/tasks/run-application/horizontal-pod-autoscale.md:441]: nit:
```suggestion
is set to 0, and the HPA's minimum replica count is greater than 0, the HPA 
stops adjusting the target (and sets the `ScalingActive` Condition on itself
```

File: content/en/docs/tasks/run-application/horizontal-pod-autoscale.md [hunk 3]
```diff
@@ -433,7 +433,14 @@ behavior:
     selectPolicy: Disabled
 ```
 
-
+## Support for implicit maintenance-mode deactivation
```
Comment (sftim) [inline content/en/docs/tasks/run-application/horizontal-pod-autoscale.md:436]:
```suggestion
## Implicit maintenance-mode deactivation
```

**Review by sftim — COMMENTED** [review]
Thanks for this tip @novas0x2a 

I've suggested some tweaks; what do you think?

**Comment by sftim** [comment]
@kubernetes/sig-autoscaling-pr-reviews happy with this revision?

**Comment by onlydole** [comment]
Thank you, @novas0x2a, and @sftim! 

/lgtm

**Review by onlydole — APPROVED** [review]
👍🏼

**Pull request merged**